lib/interface/JsonObject.dart

```diff
@@ -59,4 +59,21 @@ abstract class JsonObject {
     });
     return result;
   }
+
+  /// Returns all values as a String separated by a hyphen
+  /// value1 - value2 - value3
+  String toValueString() {
+    return toValueStringStatic(toJson());
+  }
+
+  /// Returns all values as a String separated by a hyphen
+  /// value1 - value2 - value3
+  static String toValueStringStatic(Map<String, dynamic> json) {
```

Comment:
I would just call it toValueString, static is an identifier, usually we don't put identifiers in method names.